plugins/ps/index.js

```diff
@@ -497,11 +500,21 @@ module.exports = {
         string: true,
         description: 'Healtheck endpoint for checking app readiness (web dyno only, must be valid URI /path)',
       },
+    };
+
+    const require_formation_update_option = {
+      ...require_formation_create_option,
```

Comment:
Remove healthcheck & remove command aren't options on formation create, so I moved them into an option group specific to updating.